templates/layout.html

```diff
@@ -10,7 +10,7 @@
   <meta name="description" content="{{ config.PROJECT.name }} — personal blog of Olena Yefymenko">
   <meta name="robots" content="all">
 
-  <meta name="p:domain_verify" content="8ac6cdaefda8d0a20ed08fb5c9fd3352">
+  <meta name="google-site-verification" content="5CwxblwHQ25uEaN6Nlmjr3uXxgEahbFqddRSikJviZ0" />
```

Comment:
Why is this necessary? Isn't the DNS verification enough to do the same? Also, I think that HTML 5 doesn't require a traling slash anymore.